mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/BlobStore.java

```diff
@@ -31,6 +31,8 @@
 public interface BlobStore extends Closeable {
     File get(URI blobUrl) throws IOException;
 
+    default URI getPrefix() { return URI.create(""); };
+
```

Comment:
I would rather not expose this method as it is more of an implementation detail of a specific blob store, and its unnecessary exposure increases the surface area.